### PR TITLE
Better delineation of host-based requirement sets versus Common APIs

### PR DESCRIPTION
**Pull request by AlexJerabek**
[Staging site](https://review.docs.microsoft.com/en-us/javascript/api/overview?view=excel-js-preview&branch=AlexJ-LandingPhrase)

This changes the text to avoid terms with specific meaning, such as "host-specific APIs". The goal is to properly portray the behavior of different hosts' API models, while still promoting popular content and structuring the page in an intuitive way.

Two open questions for @Rick-Kirkham and @ElizabethSamuel-MSFT:

1. Should we include Project in the list (and only link to the Common APIs)? Project is technically a supported platform, despite not having any unique requirement sets.

2. How do we best label the "Common API" card? We could leave it as is, move it under a different header, change the label to mention Office 2013, or some other solution.

### CODE REVIEW

File: docs/docs-ref-autogen/overview/overview.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ ms.date: 05/30/2020
 
 The JavaScript API for Office enables you to create web applications that interact with the object models in Office host applications. Use this section to learn more about the classes, methods, and other types available for building Office Add-ins.
 
-The following is a list of host-specific requirement sets (and the cross-host Common APIs). Each item links to a version of the API reference documentation that is supported by that requirement set (e.g. ExcelApi 1.3 shows APIs in ExcelApi 1.1, 1.2, 1.3 as well as the Common API).
+The following is a list of [requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets) for the [supported Office host applications](/office/dev/add-ins/overview/office-add-in-availability). Each item links to a version of the API reference documentation that is supported by that requirement set (e.g. ExcelApi 1.3 shows APIs in ExcelApi 1.1, 1.2, 1.3 as well as the Common APIs). Common API content includes all the [Common API requirement sets](/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets).
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/overview/overview.md:17]:
```suggestion
The following is a list of [requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets) for the [supported Office host applications](/office/dev/add-ins/overview/office-add-in-availability). Each item links to a version of the API reference documentation that is supported by that requirement set (for example, ExcelApi 1.3 shows APIs in ExcelApi 1.1, 1.2, 1.3 as well as the Common APIs). Common API content includes all the [Common API requirement sets](/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets).
```

File: docs/docs-ref-autogen/overview/overview.md [hunk 2]
```diff
@@ -137,6 +137,21 @@ The following is a list of host-specific requirement sets (and the cross-host Co
             </div>
         </a>
     </li>
+    <li>
+        <a class="card x-hidden-focus" href="/javascript/api/project?view=common-js">
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/overview/overview.md:141]:
```suggestion
        <a class="card x-hidden-focus" href="/javascript/api/common?view=common-js">
```

File: docs/docs-ref-autogen/overview/overview.md [hunk 3]
```diff
@@ -137,6 +137,21 @@ The following is a list of host-specific requirement sets (and the cross-host Co
...
+            <div class="cardImageOuter">
+                <div class="cardImage">
+                    <img src="/javascript/api/overview/images/logo-project.svg" alt="Project add-ins" />
+                </div>
+            </div>
+            <div class="cardText">
+                <h3>Project requirement sets</h3>
+                <ul>
+                    <li><a style="font-size: 1rem;" href="/javascript/api/powerpoint?view=common-js">Common APIs only</li></p>
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/overview/overview.md:150]:
```suggestion
                    <li><a style="font-size: 1rem;" href="/javascript/api/common?view=common-js">Common APIs only</li></p>
```

File: docs/docs-ref-autogen/overview/overview.md [hunk 4]
```diff
@@ -137,6 +137,21 @@ The following is a list of host-specific requirement sets (and the cross-host Co
             </div>
         </a>
     </li>
+    <li>
+        <a class="card x-hidden-focus" href="/javascript/api/project?view=common-js">
+            <div class="cardImageOuter">
+                <div class="cardImage">
+                    <img src="/javascript/api/overview/images/logo-project.svg" alt="Project add-ins" />
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/overview/overview.md:144]: Either rename the image or use the current image name...
```suggestion
                    <img src="/javascript/api/overview/images/logo-project-server.svg" alt="Project add-ins" />
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b61f4cc](https://github.com/OfficeDev/office-js-docs-reference/commits/b61f4cc3f04606f3eaab32ccfcd792a828fa1f97)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/overview/images/logo-project.svg](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/images/logo-project.svg) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/overview/overview.md](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C3%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C202006031751518867-716%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ffTcFeKZn9mXjOJx61uX35BOsatR3u5yooEZqObviBo%3D&st=2020-06-03T18%3A37%3A16Z&se=2020-07-04T18%3A42%3A16Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0e1871f2-88cb-c16b-6d81-dc80e4f66cef?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[11327f8](https://github.com/OfficeDev/office-js-docs-reference/commits/11327f8950f7c51132af877c70682fd52c736c85)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/overview/images/logo-project.svg](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/images/logo-project.svg) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/overview/overview.md](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C3%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C202006031929138303-716%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=lde%2BBgX2Pi%2BGmKprY8sT2IhjAtWCTfr05488WbuScA0%3D&st=2020-06-03T20%3A03%3A26Z&se=2020-07-04T20%3A08%3A26Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0e1871f2-88cb-c16b-6d81-dc80e4f66cef?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bd4898a](https://github.com/OfficeDev/office-js-docs-reference/commits/bd4898adc27eeeb5211ca8fae065e5911f794011)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/overview/images/logo-project.svg](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/images/logo-project.svg) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/overview/overview.md](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C3%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C202006031932385592-716%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=4F5kKUqdi91Cht%2B9iEU8ewaUy47hMRNWNYv2Vcp%2B90k%3D&st=2020-06-03T20%3A42%3A35Z&se=2020-07-04T20%3A47%3A35Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0e1871f2-88cb-c16b-6d81-dc80e4f66cef?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by AlexJerabek** [comment]
The PR is updated, though it'll take some time to rebuild.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[bf81d45](https://github.com/OfficeDev/office-js-docs-reference/commits/bf81d4508a7dc6b9ecd642b552aff4cafeec2c57)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/overview/images/logo-project.svg](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/images/logo-project.svg) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/overview/overview.md](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/overview.md) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/overview/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/toc.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/toc.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/visio/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/visio/toc.yml) | :white_check_mark:Succeeded | |
[generate-docs/scripts/postprocessor.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/generate-docs/scripts/postprocessor.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C3%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C202006032136505452-716%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yca12A6TO0omWSCFDy82hOppbhnWJC3isBeUxWxEzs8%3D&st=2020-06-03T22%3A12%3A53Z&se=2020-07-04T22%3A17%3A53Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0e1871f2-88cb-c16b-6d81-dc80e4f66cef?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1d12a24](https://github.com/OfficeDev/office-js-docs-reference/commits/1d12a2453cd23d24427e7b59dfd4231e92b998d1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/overview/images/logo-project.svg](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/images/logo-project.svg) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/overview/overview.md](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/overview.md) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/overview/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/overview/toc.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/toc.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/visio/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/docs/docs-ref-autogen/visio/toc.yml) | :white_check_mark:Succeeded | |
[generate-docs/scripts/postprocessor.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-LandingPhrase/generate-docs/scripts/postprocessor.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C3%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C202006032224248829-716%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=XqUhSea2TTm6PZ79y1Db%2BUx1Z1DRc%2BwLMc3p1z5EOfo%3D&st=2020-06-03T22%3A58%3A58Z&se=2020-07-04T23%3A03%3A58Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0e1871f2-88cb-c16b-6d81-dc80e4f66cef?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**